Library/ViewModels/MessageBannerViewModel.swift

```diff
@@ -53,14 +54,33 @@ MessageBannerViewModelInputs, MessageBannerViewModelOutputs {
     self.messageTextAlignment = bannerType.map { $0.textAlignment }
 
     self.bannerMessage = self.messageBannerConfiguration.signal.skipNil().map(second)
+    self.bannerMessageAccessibilityLabel = self.bannerMessage
+      .map { "\($0) \(Strings.Message_banner_accessibility_Double_tap_to_dismiss())" }
```

Comment:
I guess we don't have much choice here but just something to bear in mind is that string concatenation can break translations for languages where the direction isn't left to right.